packages/ui/src/components/ActionBar/actionBar.tsx

```diff
@@ -0,0 +1,28 @@
+import { Affix } from 'antd'
+import React, { ReactElement } from 'react'
+
+import { Space, SpaceProps } from '../Space'
+import { Typography } from '../Typography'
+
+const ACTION_BAR_CLASS_PREFIX = 'condo-affix'
+const AFFIX_CONTENT_WRAPPER_CLASS = `${ACTION_BAR_CLASS_PREFIX}-content-wrapper`
+
+const SPACE_SIZE: SpaceProps['size'] = [24, 24]
+
+export type ActionBarProps = {
+    message?: string
+    actions: Array<ReactElement>
+}
+
+export const ActionBar: React.FC<ActionBarProps> = (props) => {
+    const { actions, message } = props
```

Comment:
You can declare a minimum length of this prop with that construction: `actions: [ReactElement, ...ReactElement[]]`